### PR TITLE
rephrase optional step for datadog metrics

### PR DESCRIPTION
**Pull request by afastronomer**

### CODE REVIEW

File: astro/deployment-metrics.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ Astro does not export any [service checks](https://docs.datadoghq.com/integratio
    - **Key:** `DATADOG_API_KEY`
    - **Value:** `<Your-Datadog-API-key>`.
    Select the **Secret?** checkbox. This ensures that your Datadog API key is saved securely and is not available to Workspace users in plain text.
-4. Default Datadog site specified in Astro is **datadoghq.com**. Only if you use different one, extra variable has to be created:
+4. Optional. If you don't want to use the default Datadog site (datadoghq.com) defined by Astronomer, add the following environment variable:
```
Comment (stuart23) [inline astro/deployment-metrics.md:190]:
```suggestion
4. Optional. If you use a Datadog site that is not the default (datadoghq.com), add the following environment variable:
```

**Review by stuart23 — CHANGES_REQUESTED** [review]
Just want to clarify that it isn't up to what the customer "wants", but rather what is configured in their DD account.

**Comment by stuart23** [comment]
Reference if it is relevant to help describe: https://docs.datadoghq.com/getting_started/site/

File: astro/deployment-metrics.md [hunk 2]
```diff
@@ -187,7 +187,7 @@ Astro does not export any [service checks](https://docs.datadoghq.com/integratio
    - **Key:** `DATADOG_API_KEY`
    - **Value:** `<Your-Datadog-API-key>`.
    Select the **Secret?** checkbox. This ensures that your Datadog API key is saved securely and is not available to Workspace users in plain text.
-4. Default Datadog site specified in Astro is **datadoghq.com**. Only if you use different one, extra variable has to be created:
+4. Optional. If you use a Datadog site that is not the default (datadoghq.com), add the following environment variable:
```
Comment (kwcanuck) [inline astro/deployment-metrics.md:190]:
```suggestion
4. Optional. If your organization doesn't use the default Datadog site datadoghq.com, add the following environment variable:
```

**Review by stuart23 — APPROVED** [review]
Robs wording sounds much better.

**Pull request merged**